features/rspec.feature

```diff
@@ -0,0 +1,17 @@
+Feature: The rspec command
+
+  Background:
+    Given an empty file named "spec/spec_helper.rb"
```

Comment:
Please do not explicitly create empty files here, but create them with a short hint of their intention instead.

```suggestion
    Given a file named "spec/spec_helper.rb" with "enable rspec"
```
(or whatever the reason is why you need this file)

Cucumber is an abstract language and should describe expected behavior (there is a file). It should not be used to "program" (like creating an empty file because its content does not matter).